coalesce/config/common.py

```diff
@@ -204,5 +204,8 @@ class Common(Configuration):
         'DEFAULT_AUTHENTICATION_CLASSES': (
             'rest_framework.authentication.SessionAuthentication',
             'rest_framework.authentication.TokenAuthentication',
+            'rest_framework_simplejwt.authentication.JWTAuthentication'
         )
     }
+    
```

Comment:
flake8 is complaining about this blank line containing whitespace. This change came from my commit which has since been merged into master anyway. The version that was merged doesn't have the whitespace that causes the flake8 test to fail, so you can git rebase on master to get rid of this issue.

Reply:
Sweet, I'll update that now.